snapshot-labs/sx-monorepo
# fix(ui): do not repeat space cover image

### Summary



This PR fixes the fallback space/user cover image, which is being repeated instead of being "covered".

This also center the image (optional, I feel it's more consistent, but it may be too bright)

Before fix

![Screenshot 2024-08-16 at 16 49 35](https://github.com/user-attachments/assets/2d71c531-3bc5-4dd8-8996-2ecb349206fa)

After fix

![Screenshot 2024-08-16 at 16 44 52](https://github.com/user-attachments/assets/65e24301-b3ee-4659-bd2e-e95cc80a5e81)

Version without the image being centered

![Screenshot 2024-08-16 at 16 44 24](https://github.com/user-attachments/assets/432a1aa8-08e7-4f3f-94d9-ff396709b669)

This PR also moves from `backdrop-filter` over the image to direct `blur` effect on the image, avoiding the lag on first page load when applying the blur filter

https://github.com/user-attachments/assets/fc99aa8a-1a37-4481-aff6-becf1002effb

This also fix the blur effect flashing when being recomputed, on page resize

https://github.com/user-attachments/assets/1a6d52af-5d71-40f3-ae93-9ecff14cca8d




### How to test

1. Go to the /explore page
2. Inspect the element, and remove the blur effect on the space cover
3. you should see the default avatar being shown as "covered", instead of being repeated
4. Go to any space overview page
5. It should show the space cover, without any lag when applying the blur filter
6. Resizing the page should not "flash" the blur effect
